lib/editor/atto/plugins/kalturamedia/yui/build/moodle-atto_kalturamedia-button/moodle-atto_kalturamedia-button.js

```diff
@@ -15,194 +15,168 @@ YUI.add('moodle-atto_kalturamedia-button', function (Y, NAME) {
-                containter.append(iframe);
-                containter.append(embedButton);
-                // Set the dialogue content, and then show the dialogue.
-                dialogue.set('bodyContent', containter)
-                    .show();
-            },
-
-            _getIframeURL: function() {
-
-                var args = Y.mix({
-                        elementid: this.get('host').get('elementid'),
-                        contextid: this.get('contextid'),
-                        height: '600px',
-                        width: '1112px'
-                    },
-                    this.get('area'));
-                return M.cfg.wwwroot + '/lib/editor/atto/plugins/kalturamedia/ltibrowse.php?' +
-                    Y.QueryString.stringify(args);
-            },
-
-            _getCourseId: function() {
-                var courseId;
-                var bodyClasses = document.getElementsByTagName('body')[0].className;
-                var classes = bodyClasses.split(' ');
-                for(i in classes)
-                {
-                    if(classes[i].indexOf('course-') > -1)
-                    {
-                        var parts = classes[i].split('-');
-                        courseId = parts[1];
-                    }
-                }
-
-                return courseId;
-            },
-
-            _replaceKafUriWithToken: function(sourceUrl, kafUri, kalturaToken) {
-                sourceUrl = this._removeProtocolFromUrl(sourceUrl);
-                kafUri = this._removeProtocolFromUrl(kafUri);
-                return sourceUrl.replace(kafUri, kalturaToken);
-            },
-
-            _removeProtocolFromUrl: function(fullUrl) {
-                return fullUrl.replace(/^https?:\/\//,'');
-            },
-
-            embedItem: function(what) {
-                var dialogue = this.getDialogue({
-                    focusAfterHide: null
-                });
-
-                data = Y.one('#KalturaMediaSubmit')._getDataAttributes();
-                embedInfo = {};
-                for(param in data)
+                    width: '1112px'
+                },
+                this.get('area'));
+        return M.cfg.wwwroot + '/lib/editor/atto/plugins/kalturamedia/ltibrowse_container.php?' +
+                Y.QueryString.stringify(args);
+    },
+    
+    _getCourseId: function() {
+        var courseId;
+        var bodyClasses = document.getElementsByTagName('body')[0].className;
+        var classes = bodyClasses.split(' ');
+        for(i in classes)
+        {
+            if(classes[i].indexOf('course-') > -1)
                 {
-                    var isEmbedInfo = param.split('-');
-                    if(isEmbedInfo[0] == 'embedinfo')
-                    {
-                        embedInfo[isEmbedInfo[1]] = data[param];
-                    }
+                    var parts = classes[i].split('-');
+                    courseId = parts[1];
```

Comment:
@muli who says parts[1] exists, what if the input string is 'course-' ? (is it possible ?)